### ISSUE TITLE
Separate native WooCommerce users from BBS associated users

### ISSUE THREAD

**Issue by israeldeweb** [deweb-io/premium#11]
Currently, if someone registers to subbscribe.com and chooses an existing blockchain ID as a username, it will fuck up our system. If he uses a matching email (`${blockchainId}@bbs.netword}` at the moment), then the matching BBS user can take over his account. If he uses another email, the BBS user will not be able to use the plugin (we will try and fail to create a user for him on every attempt).

In short, we need to make it impossible for subbscribe.com users to create BBS related users. And think a bit more of specifically how we combine them (I hate the email trick, for example).
